edgelet/edgelet-docker/src/config.rs

```diff
@@ -90,6 +91,20 @@ impl DockerConfig {
     }
 }
 
+pub const UPSTREAM_PARENT_KEYWORD: &str = "$upstream";
```

Comment:
`"$upstream/"` ? So as to not match `"$upstreamhahaprankd/..."`

But I see the original code also had this, so if you want to leave it as it is that's fine.

Reply:
@huguesBouvier wrote it originally. I'm just preserving functionality :)